js/src/containers/TwitterEditor.js

```diff
@@ -93,7 +89,7 @@ export default compose( [
 			imageWarnings: getTwitterWarnings(),
 			authorName: getAuthorName(),
 			siteUrl: getSiteUrl(),
-			isPremium: !! isPremium,
+			isPremium: !! window.wpseoAdminL10n.isPremium,
```

Comment:
Here `isPremium` is read from `window.wpseoAdminL10n`. In other places it's read from `getL10nObject` which gets it's data from `window.wpseoScriptData.metabox`. That feels like either duplication or an error.

Reply:
Removed the `wpseoAdminL10n` variant and its uses in Yoast SEO. Also checked usage in Premium and found only `getL10nObject` uses.